From 488f9fb1e4b5bd584e4041c9476d902ea3ea2c5d Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Wed, 8 Jan 2025 11:43:06 +0100
Subject: [PATCH] arm64: dts: st: correct imx335 reset gpio polarity in
 stm32mp257f-ev1.dts

With commit 9eac174a9b89 ("media: imx335: Fix reset-gpio handling")
introduction, reset gpio line must be corrected within the device-tree
in order to match with expected behavior of the reset of module.

Change-Id: I927ef1cb9054e37302a255be64f5c1b4f593c1a2
Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -446,7 +446,7 @@
 		avdd-supply = <&imx335_2v9>;
 		ovdd-supply = <&imx335_1v8>;
 		dvdd-supply = <&imx335_1v2>;
-		reset-gpios = <&gpioi 7 (GPIO_ACTIVE_HIGH | GPIO_PUSH_PULL)>;
+		reset-gpios = <&gpioi 7 (GPIO_ACTIVE_LOW | GPIO_PUSH_PULL)>;
 		powerdown-gpios = <&gpioi 0 (GPIO_ACTIVE_HIGH | GPIO_PUSH_PULL)>;
 		status = "okay";
 
